src/InstanceService.ts

```diff
@@ -397,6 +397,26 @@ export class InstanceService {
     }
   }
 
+  async sendButtonAction(
+    instanceId: string,
+    conversationId: string,
+    referenceMessageId: string,
+    buttonId: string,
+  ): Promise<string> {
+    const instance = this.getInstance(instanceId);
+    const service = instance.account.service;
```

Comment:
Is it possible that instance is undefined?

Reply:
At this stage not. Otherwise tsc would also fail to compile.

Reply:
Ah okay, it just looks like instanceId is coming from the outside. In other words it seems to be guaranteed that an instance exists for the given instanceId. Alright :)